packages/webapp/constants/types.ts

```diff
@@ -11,10 +11,10 @@ export const enum CHAIN_IDS {
   BASE_SEPOLIA = 84532
 }
 
-export const ALCHEMY_RPC_URLS: Map<number, string> = new Map([
-  [CHAIN_IDS.ETHEREUM, "https://eth-mainnet.g.alchemy.com/v2/"],
-  [CHAIN_IDS.BASE, "https://base-mainnet.g.alchemy.com/v2/"]
-])
+export const ALCHEMY_RPC_URLS= {
```

Comment:
Typescript likes this syntax better because `map.get(X)` can potential return `undefined`